doc/tutorials/pick_and_place_with_moveit_task_constructor/pick_and_place_with_moveit_task_constructor.rst

```diff
@@ -10,8 +10,8 @@ Pick and Place with MoveIt Task Constructor
 
 This tutorial will walk you through creating a package that plans a pick and place operation using `MoveIt Task Constructor <https://github.com/ros-planning/moveit_task_constructor/tree/ros2/>`_. MoveIt Task Constructor provides a way to plan for tasks that consist of multiple different subtasks (known as stages).
 
-Basic Concepts
---------------
+1 Basic Concepts
```

Comment:
The [PickNik repo](https://moveit.picknik.ai/humble/doc/tutorials/pick_and_place_with_moveit_task_constructor/pick_and_place_with_moveit_task_constructor.html) has this section removed which I think cleans the intro up nicely. I'm not sure if we should PR that branch in first or just make those changes with this PR. Either way I'll have to update this file with new numbering if we decide to move the Basic Concepts section out of this page

Reply:
@stephanie-eng Please share your thoughts on this

Reply:
The consolidation of the MTC concepts and tutorials was done as part of https://github.com/ros-planning/moveit2_tutorials/pull/393. This hasn't been ported to the Humble branch, but you should be able to see the most recent stuff on the website if you change the tutorial version to main (Rolling).

You have a lot of nice fixes and clarifications here - let's get these merged first and deal with any restructuring in a separate PR.